examples/stable_baselines3/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM tensorflow/tensorflow:2.4.0-gpu
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Prevent tzdata from trying to be interactive
+ENV TZ=Europe/Minsk
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# http://bugs.python.org/issue19846
+# > At the moment, setting "LANG=C" on a Linux system *fundamentally breaks Python 3*, and that's not OK.
+ENV LANG C.UTF-8
+
+# Install libraries
+RUN apt-get update --fix-missing && \
+    apt-get install -y \
+        software-properties-common && \
+    add-apt-repository -y ppa:deadsnakes/ppa && \
+    add-apt-repository -y ppa:sumo/stable && \
+    apt-get update && \
+    apt-get install -y \
+        libspatialindex-dev \
+        python3.7 \
+        python3.7-dev \
+        python3.7-venv \
+        git \
+        vim \
+        wget \
```

Comment:
Are `git`, `vim`, `wget`, needed?

Reply:
ah, git and vim were for my debugging purposes, ill remove them, although wget was from https://github.com/huawei-noah/SMARTS/blob/master/utils/docker/Dockerfile which I based this off of

Reply:
Dockerfile has been edited appropriately.